core/src/main/resources/db/migration/V26__add_uploaded_date.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `Package`
+ADD COLUMN uploadedAt DATETIME(3) NOT NULL DEFAULT 0;
```

Comment:
It's what we use for other DateTime columns. I'll double check they are instants.